components/brave_rewards/resources/page/promos/index.tsx

```diff
@@ -4,14 +4,16 @@
 
 import * as React from 'react'
 import tapBg from './assets/tap_bg.svg'
-import { StyledLink, StyledInfo } from '../../ui/components/sidebarPromo/style'
+import upholdBg from './assets/uphold_bg.png'
+import { StyledInfo } from '../../ui/components/sidebarPromo/style'
 import { getLocale } from '../../../../common/locale'
 
-export type PromoType = 'tap-network'
+export type PromoType = 'uphold' | 'tap-network'
 
 export interface Promo {
   title: string,
   imagePath: string,
+  link: string
```

Comment:
Should we just remove the other commas in this interface while we're here?

Reply:
removed